index.js

```diff
@@ -62,7 +63,28 @@ function initConfig(projectConfig) {
       minorVersion: process.env.GAE_MINOR_VERSION
     }
   };
-  var config = extend(true, {}, require('./config.js'), projectConfig, envConfig);
+
+  var envSetConfig = {};
+  if (process.env.hasOwnProperty('GCLOUD_TRACE_CONFIG')) {
+    try {
+      envSetConfig = require(path.resolve(process.env.GCLOUD_TRACE_CONFIG));
+    } catch (error) {
+
+      var logger = common.logger({
+        level: common.logger.LEVELS[2],
+        tag: '@google-cloud/trace-agent',
+      });
+
+      logger.error('The environment variable GCLOUD_TRACE_CONFIG was set,' +
```

Comment:
That actually makes sense, and I agree with you, I rather have it vomit in trying to start the application/container so that its super obvious.  I'll update it shortly.